.github/workflows/e2e.yml

```diff
@@ -17,11 +17,6 @@ jobs:
     steps:
       - name: Check out the repository
         uses: actions/checkout@v2
-        with:
```

Comment:
We added it because otherwise the shipyard scripts get a little bit confused about tags, but
may be it's not necessary for E2E, and just for releasing/tagging images?... let's see how it goes

Reply:
Yeah, I think that's the case. It's hard to imagine how there could be an issue with removing this from the E2E workflow that wouldn't already have shown up.